### ISSUE TITLE
Can you please update this project 

### ISSUE THREAD

**Issue by oscarlagatta** [gothinkster/react-redux-realworld-example-app#143]
Last commit was on 29 Nov 2018, seriously ? Please update. It's very difficult to follow the React training, really difficult. React router, etc etc is far behind with versions. Please update.

**Comment by zenzjtech** [comment]
Yes, I don't know what is happening with Thinkster.io, but many things will need to be update.
For example, The react-router API has been changed, corresponding changes'd need to be made in this tutorial.
https://thinkster.io/tutorials/react-router/setting-up-the-routing-structure

**Comment by devellopah** [comment]
The other popular options(vue, angular, elm, svelter) are up to date. Even react/mobx is up to date. But the most popular is abandoned. The fact confuses me a lot...

**Comment by sergioc32** [comment]
Please update!

**Comment by sohag033** [comment]
Please update the project!!!!!

**Comment by Allianzcortex** [comment]
I have pushed a new version with React FC + Typescript +  Redux + Hooks + Immer 

repo : https://github.com/Allianzcortex/react-typescript-hooks-realworld

link : https://react-typescript-hooks-realworld.vercel.app/

demo : 
![image](https://user-images.githubusercontent.com/7290780/121895388-8a585d00-ccf6-11eb-8b3d-032163ddb627.png)

Hope it can be useful ( - :
